src/tools/scripts/test_template_changes.sh

```diff
@@ -61,7 +61,7 @@ if [ "${NUM_DIFFS}" -ne "0" ]; then
   # Weird syntax is to make is Mac compatible: https://stackoverflow.com/a/1252191/2144578
   ESCAPED_OUTPUT=$(sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/%0A/g' -e 's/\r/%0D/g' -e 's/\%/%25/g' "${DIFF_FILENAME}")
   PR_COMMENT="Please note, that the following diffs happen in the templates on this branch compared to main:%0A\`\`\`%0A${ESCAPED_OUTPUT}%0A\`\`\`%0A"
-  echo "::set-env name=PR_COMMENT::${PR_COMMENT}"
+  echo "::set-output name=PR_COMMENT::${PR_COMMENT}"
```

Comment:
Do we want to just set the output here or store in the `$GITHUB_ENV`?

Reply:
I thought output would be better as it would prevent any name clashes (PR_COMMENT isn't that unique and not hard to assume we might use that again) but after spending far too long trying to get it to work with escaping line numbers I've given up and gone back to env variables. It seems github script doesn't like outputs with line breaks.